bugcatcher/ftl.py

```diff
@@ -23,7 +24,7 @@
 # from Login import current_login
 
 args = None
-default_extensions = ['.sol', '.json', '.txt', '.py']
+default_extensions = []
```

Comment:
Do we want to have the default be none? What about the case where the project isn't in git?